core/templates/pages/admin-page/admin-page.constants.ts

```diff
@@ -23,6 +23,10 @@ export const AdminPageConstants = {
   ADMIN_TOPICS_CSV_DOWNLOAD_HANDLER_URL: '/admintopicscsvdownloadhandler',
 
   ADMIN_JOB_OUTPUT_URL_TEMPLATE: '/adminjoboutput?job_id=<jobId>',
+  MEMORY_CACHE_HANDLER_URL: '/memorycacheadminhandler',
```

Comment:
Add these in alphabetically sorted order.

Reply:
done!